server/views/pages/goal/update/review.njk

```diff
@@ -42,7 +50,7 @@
               text: "Note"
             },
             value: {
-              text: data.note
+              text: data.notes
```

Comment:
A couple of the fields in the DTO have slightly different names to the form object - hence this change to `notes` and the one below to `sequenceNumber`